exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -29,6 +32,14 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/log"
 )
 
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Channel: subscription.TickerChannel, Asset: asset.All},
+	{Enabled: true, Channel: subscription.AllTradesChannel, Asset: asset.All},
+	{Enabled: true, Channel: subscription.CandlesChannel, Asset: asset.Spot, Interval: kline.OneMin},
```

Comment:
Same approach as other exchanges: Just remove the spot pairs from the overlapping margin sub.
`$.AssetPairs` only contains the relevant pairs for this subscription, so in the case of Candles it'll only have one of margin or spot. For Ticker and Orderbook it will have both.

Since I'm changing unreleased defaults, Don't forget to purge your local config subscriptions to get the new defaults.

Fixed 991af425f